bundles/net.tourbook/src/net/tourbook/ui/views/rawData/DialogMergeTours.java

```diff
@@ -1426,7 +1382,6 @@ private void onModifyProperties() {
       _tourChartConfig.isRelativeValueDiffScaling = _chkValueDiffScaling.getSelection();
 
       if (_chkPreviewChart.getSelection()) {
-         // update chart
```

Comment:
I put it back. But i removed it initially because the line just under it says the exact same thing, it seems very redundant to me.
![image](https://user-images.githubusercontent.com/25235665/151847712-b16c1b0f-d0a0-40f2-9591-cd15e55c84af.png)

Reply:
Sometimes when I create a new if statement, firstly I put comments into the if/else case before putting code also into it, or as pointed out above I only read the comments when I try to understand the code